docs/api-reference/endpoints/certificate-templates/get-by-id.mdx

```diff
@@ -0,0 +1,4 @@
+---
+title: "Get by ID"
```

Comment:
Change title to `Retrieve` to be consistent with other PKI endpoint titles (current convention is List, Retrieve, Create, Update, Delete).